search-enterprise/latest/admin/index.md

```diff
@@ -371,6 +371,48 @@ select to_char(to_timestamp(min(commit_time_ms)/1000),'YYYYMMDDHHMI') as fromTim
 
 Because it is easier to have **unbalanced partitions** when indexing by date range, it is recommended to use this strategy only when we are interested in indexing a specific date range, while it is recommended to perform a reindex by id range when we need to perform a **full reindex**.
 
```

Comment:
The  "AWS read replicas" link is broken, page does not exist anymore.

Reply:
Not a part of this review. Covered in another PR.